src/Umbraco.Web.UI.Client/src/less/pages/login.less

```diff
@@ -50,7 +52,7 @@
 }
 
 .login-overlay .form {
-    position:fixed;
+    position:relative;
```

Comment:
Changed to `position: relative;` and set container to `overflow-y: auto;` so it overflow, you are able to scroll dow, e.g. on narrow screens or with keypad open.